app/components/provider_interface/conditions_component.html.erb

```diff
@@ -1,20 +1,26 @@
 <table class="govuk-table govuk-!-margin-bottom-2">
   <tbody class="govuk-table__body">
-    <% condition_rows.each do |row| %>
+    <% if conditions.none? %>
       <tr class="govuk-table__row conditions-row">
-        <td class="govuk-table__cell"><%= row %></td>
-        <td class="govuk-table__cell govuk-table__cell--numeric">
-          <% if conditions.present? %>
-            <% if conditions_met? %>
-              <%= govuk_tag(text: 'Met', colour: 'green') %>
-            <% elsif known_conditions_state? %>
-              <%= govuk_tag(text: 'Not met', colour: 'red') %>
-            <% else %>
-              <%= govuk_tag(text: 'Pending', colour: 'grey') %>
-            <% end %>
-          <% end %>
+        <td class="govuk-table__cell">
+          No conditions have been set for this offer.
         </td>
       </tr>
-    <% end %>
+    <% else %>
+        <% conditions.each do |condition| %>
+          <tr class="govuk-table__row conditions-row">
+            <td class="govuk-table__cell"><%= condition.text %></td>
+            <td class="govuk-table__cell govuk-table__cell--numeric">
+              <% if condition.met? %>
+                <%= govuk_tag(text: 'Met', colour: 'green') %>
+              <% elsif condition.unmet? %>
+                <%= govuk_tag(text: 'Not met', colour: 'red') %>
+              <% else %>
+                <%= govuk_tag(text: 'Pending', colour: 'grey') %>
+              <% end %>
```

Comment:
Just testing this locally in the deferred offer flow. This component is used when previewing a reinstated deferred offer. So this if condition needs to show the new condition status rather than the current one

Reply:
Yes got the same 👍